src/compiler/scala/tools/nsc/typechecker/Implicits.scala

```diff
@@ -1585,6 +1590,7 @@ trait ImplicitsStats {
   val improvesCachedCount = newSubCounter("  #implicit improves cached ", implicitSearchCount)
   val inscopeImplicitHits = newSubCounter("  #implicit inscope hits", implicitSearchCount)
   val oftypeImplicitHits  = newSubCounter("  #implicit oftype hits ", implicitSearchCount)
+  val viewsNanos          = newSubTimer  ("time spent in views", typerNanos)
```

Comment:
I think this description is a bit misleading: `allViewsFrom` seems to be used only by scaladoc, the compiler uses `inferImplicit` with `isView = true`.

Reply:
Intersting, didn't know this. Then I'm removing it.